packages/mux-video/src/index.ts

```diff
@@ -291,7 +294,14 @@ class MuxVideoBaseElement extends CustomVideoElement implements Partial<MuxMedia
   }
 
   get playbackId(): string | undefined {
-    return this.getAttribute(Attributes.PLAYBACK_ID) ?? undefined;
+    if (this.hasAttribute(Attributes.PLAYBACK_ID)) {
+      return this.getAttribute(Attributes.PLAYBACK_ID) as string;
+    }
+
+    return toPlaybackIdFromSrc(this.src) ?? undefined;
+    // return isMuxVideoSrc(this)
+    //   ? toPlaybackIdFromSrc(this.src)
+    //   : undefined;
```

Comment:
can be removed?

Reply:
ah actually we may want to use the commented out code. Didn't end up being necessary for the issue I encountered, but it's a safer way to resolve. Thoughts?

Reply:
oh never mind I remember why I removed it. There was a circular def. Deleting and can revisit later, re: "safer way to resolve playback id"